doc/intro.md

```diff
@@ -84,22 +86,33 @@ Three alternatives are available to produce population projections:
 
```

Comment:
I still think this diagram will confuse people- can you find a way to label or describe what it is supposed to be showing? I think it shows which data sources feed into the future rates for each component, which then feed into the forward projection?